eslint/babel-eslint-plugin/src/rules/no-undef.cjs

```diff
@@ -1,7 +1,9 @@
 const ruleComposer = require("eslint-rule-composer");
 const eslint = require("eslint");
 
-const rule = new eslint.Linter().getRules().get("no-undef");
+const rule = new eslint.Linter({ configType: "eslintrc" })
+  .getRules()
+  .get("no-undef");
```

Comment:
I didn't mark the original issue as fixed because I thought this was just a temporary solution.